tests/integration/test_vulnerability_detector/test_scan_results/test_scan_vulnerabilities_triaged_null.py

```diff
@@ -0,0 +1,192 @@
+def test_scan_triaged_null_vulnerabilities(configuration, metadata, agent_system, set_wazuh_configuration_vdt,
+                                           truncate_monitored_files, clean_cve_tables_func, prepare_scan,
+                                           restart_modulesd_function):
+    '''
+    description: Check if a vulnerable package with triaged NULL, is detected with the partial scan.
+
+    test_phases:
+        - Set a custom Wazuh configuration.
+        - Mock an agent.
+        - Force a full scan.
+        - Restart wazuh-modulesd.
+        - Insert a vulberable package with triaged NULL.
+        - Check that after partial scan triaged changes to 1.
+
+    wazuh_min_version: 4.4.0
+
+    tier: 1
+
+    parameters:
+        - configuration:
+            type: dict
+            brief: Configuration loaded from `configuration_template`.
+        - metadata:
+            type: dict
+            brief: Test case metadata.
+        - agent_system:
+            type: str
+            brief: System to set to the mocked agent.
+        - set_wazuh_configuration_vdt:
+            type: fixture
+            brief: Set wazuh configuration.
+        - truncate_monitored_files:
+            type: fixture
+            brief: Truncate all the log files and json alerts files before and after the test execution.
+        - clean_cve_tables_func:
+            type: fixture
+            brief: Clean all CVE tables.
+        - prepare_scan:
+            type: fixture
+            brief: Setup the initial test state.
+        - restart_modulesd_function:
+            type: fixture
+            brief: Restart wazuh-modulesd daemon before starting a test, and stop it after finishing.
+
+    assertions:
+        - Verify that the log a partial scan will be run on agent appears in ossec.log.
+        - Verify that the triaged field of sys_programs table has changed to 1.
+
+    input_description:
+        - The `configuration_scan_vulnerabilities_triaged_null.yaml` file provides the module configuration for this
+          test.
+        - The `cases_scan_vulnerabilities_triaged_null.yaml` file provides the test cases.
+
+    expected_output:
+        - f"A partial scan will be run on agent '{agent_id}'"
+    '''
+    agent_id = prepare_scan
+
+    # Insert mocked vulnerables packages.
+    agent_db.insert_package(name=vd.VULNERABLE_PACKAGES[1]['name'], version=vd.VULNERABLE_PACKAGES[1]['version'],
+                            source=vd.VULNERABLE_PACKAGES[1]['name'], agent_id=agent_id,
+                            vendor='Red Hat, Inc.', triaged=metadata['triaged'])
+
+    # Update packages sync status.
+    agent_db.update_sync_info(agent_id=agent_id, component="syscollector-packages")
+
+    # Check for the next partial scan
+    evm.check_partial_scan_start_log(agent_id=agent_id, timeout=vd.T_15)
+
+    query_string = f"agent {agent_id} sql SELECT triaged FROM sys_programs"
+
+    query_value = query_wdb(query_string)[0]
```

Comment:
Encapsulate this in a framework method

Reply:
Done in [here](https://github.com/wazuh/wazuh-qa/pull/3587/commits/840de2cd5fb93a74e75cb6bbcb6511b3ec15491b)